defaults/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-postgresql_support_libpq_version: "9.5.*.pgdg14.04+2"
+postgresql_support_libpq_version: "9.5.*.pgdg*"
```

Comment:
They are separate because `azavea.postgresql` role and the `azavea.postgresql-support` role usually don't get combined on that same machine. The latter is meant to install dependencies required to use the PostgreSQL Ansible modules to create users, databases, extensions, etc.

`postgresql_support_libpq_version` can also be tweaked in `group_vars/all` to avoid having to update this role out-of-band.